var/spack/repos/builtin/packages/py-kornia-rs/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyKorniaRs(PythonPackage):
+    """Low level implementations for computer vision in Rust."""
+
+    homepage = "https://www.kornia.org/"
```

Comment:
Are you sure this site supports the `https` protocol?  I can only access it via `http`.

Reply:
http is the right one

Reply:
Changed to http. I just copied the https from [here](https://github.com/kornia/kornia/blob/d6b18a5188f3ee09447a542306c86d1c71ecf2f6/pyproject.toml#L43). This should either be changed or support for https should be added.